index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav class="nav_bar">
+      <h1>Original Trombones</h1>
+      <ul class="nav_list">
+        <li class="nav_link">
+          <a href="#">Features</a>
+        </li>
+        <li class="nav_link">
+          <a href="#">How it works</a>
+        </li>
+        <li class="nav_link">
+          <a href="#">Pricing</a>
+        </li>
+      </ul>
+    </nav>
```

Comment:
Kindly use a header tag to put the h1 and nav inside since it's better semantically.